composer.lock

```diff
@@ -3831,16 +3860,16 @@
         },
         {
             "name": "drupal/core-recommended",
-            "version": "8.9.13",
+            "version": "8.9.14",
```

Comment:
This should bring the update to pear/archive_tar to 1.4.13, but it doesn't. Confused as to why not. (And 1.4.12 has a security update required https://github.com/UN-OCHA/iasc8/security/dependabot).

`composer clearcache` didn't help - any other suggestions?